proposals/simd/BinarySIMD.md

```diff
@@ -27,143 +27,143 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
+| `i16x8.eq`                |    `0x4a`| -                  |
+| `i32x4.eq`                |    `0x4c`| -                  |
+| `f32x4.eq`                |    `0x4d`| -                  |
+| `f64x2.eq`                |    `0x4e`| -                  |
+| `i8x16.ne`                |    `0x4f`| -                  |
+| `i16x8.ne`                |    `0x50`| -                  |
+| `i32x4.ne`                |    `0x51`| -                  |
+| `f32x4.ne`                |    `0x53`| -                  |
+| `f64x2.ne`                |    `0x54`| -                  |
+| `i8x16.lt_s`              |    `0x55`| -                  |
+| `i8x16.lt_u`              |    `0x56`| -                  |
+| `i16x8.lt_s`              |    `0x57`| -                  |
+| `i16x8.lt_u`              |    `0x58`| -                  |
+| `i32x4.lt_s`              |    `0x59`| -                  |
+| `i32x4.lt_u`              |    `0x5a`| -                  |
+| `f32x4.lt`                |    `0x5d`| -                  |
+| `f64x2.lt`                |    `0x5e`| -                  |
+| `i8x16.le_s`              |    `0x5f`| -                  |
+| `i8x16.le_u`              |    `0x60`| -                  |
+| `i16x8.le_s`              |    `0x61`| -                  |
+| `i16x8.le_u`              |    `0x62`| -                  |
+| `i32x4.le_s`              |    `0x63`| -                  |
+| `i32x4.le_u`              |    `0x64`| -                  |
+| `f32x4.le`                |    `0x67`| -                  |
+| `f64x2.le`                |    `0x68`| -                  |
+| `i8x16.gt_s`              |    `0x69`| -                  |
+| `i8x16.gt_u`              |    `0x6a`| -                  |
+| `i16x8.gt_s`              |    `0x6b`| -                  |
+| `i16x8.gt_u`              |    `0x6c`| -                  |
+| `i32x4.gt_s`              |    `0x6d`| -                  |
+| `i32x4.gt_u`              |    `0x6e`| -                  |
+| `f32x4.gt`                |    `0x71`| -                  |
+| `f64x2.gt`                |    `0x72`| -                  |
+| `i8x16.ge_s`              |    `0x73`| -                  |
+| `i8x16.ge_u`              |    `0x74`| -                  |
+| `i16x8.ge_s`              |    `0x75`| -                  |
+| `i16x8.ge_u`              |    `0x76`| -                  |
+| `i32x4.ge_s`              |    `0x77`| -                  |
+| `i32x4.ge_u`              |    `0x78`| -                  |
+| `f32x4.ge`                |    `0x7b`| -                  |
+| `f64x2.ge`                |    `0x7c`| -                  |
+| `f32x4.neg`               |    `0x7d`| -                  |
+| `f64x2.neg`               |    `0x7e`| -                  |
+| `f32x4.abs`               |    `0x7f`| -                  |
+| `f64x2.abs`               |    `0x80`| -                  |
+| `f32x4.min`               |    `0x87`| -                  |
+| `f64x2.min`               |    `0x88`| -                  |
```

Comment:
This part of the opcode space is mostly float-only operations. Would it make sense to reserve opcodes for integer min/max elsewhere? It could be similar to add, sub, and mul. Alternatively, we could reorganize to put floating point arithmetic ops next to their integer counterparts.

Reply:
That makes sense, I mostly just ported an old pull request to the new space. I could go ahead and re-organize so the floating point arithmetic ops are next to the integer ones, or just move the sprinkled in Floating point ops to the end (which is somewhat consistent with other Wasm opcodes - the separation between integer/floating point opcodes). Alternatively I would prefer to order these according to type - mostly because some types have operations that are not supported on other types, and it makes the overall organization cleaner IMO. WDYT?

Reply:
Moving all fp ops to the end sounds good. I generally prefer ordering by operation over ordering by type, though.

Reply:
The MVP instructions are grouped first by what kind of instruction (unary, binary, conversion, etc.) then by type. It would be nice to do the same here, if we could.